mkdocs.yml

```diff
@@ -6,16 +6,16 @@ site_author: 'Michael Johanson'
 site_url: 'https://edgexfoundry.github.io/edgex-docs/'
 repo_url: 'https://github.com/edgexfoundry/edgex-go'
 repo_name: 'edgex/edgex-go'
-copyright: 'Copyright &copy; 2020 EdgeX Foundry'
-strict: true
+copyright: 'Copyright &copy; 2022 EdgeX Foundry'
+strict: false   # mkdocs-materials will prevent running on 0.0.0.0 otherwise (or have custom image)
```

Comment:
Comment is not clear to me. 
If strict=true then mkdocs-materials will prevent running on 0.0.0.0 otherwise (or have custom image)??
and what is `custom image` referring to?
What is does the strict setting effect?